generators/app/templates/Index.jsx

```diff
@@ -1,20 +1,32 @@
-/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
 import React from 'react';
 import PropsTable from 'terra-props-table';
 import Markdown from 'terra-markdown';
-import ReadMe from '<%= projectName %>/docs/README.md';
-import { version } from '<%= projectName %>/package.json';
+import IndexTemplate from 'terra-site/src/IndexPageTemplate';
+
+import ReadMe from '<%= srcPath %>/docs/README.md';
+import { version } from '<%= srcPath %>/package.json';
 
 // Component Source
 // eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
-import <%= projectClassName %>Src from '!raw-loader!<%= projectName %>/src/<%= projectClassName %>';
+import <%= projectClassName %>Src from '!raw-loader!<%= srcPath %>/src/<%= projectClassName %>';
 
 // Example Files
+import Default<%= projectClassName %> from './<%= importPath %>Default<%= projectClassName %>';
+import Default<%= projectClassName %>Src from '!raw-loader!./<%= importPath %>Default<%= projectClassName %>.jsx';
 
 const <%= projectClassName %>Examples = () => (
   <div>
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
+
+    <h1 style={{ paddingBottom: '0.3em', borderBottom: '1px solid #eaecef' }}>Examples</h1>
```

Comment:
Can we use one of our terra heading elements as part of our generator now? Feels like the best way to get users to use our components is to build our example sites with them. https://terra-core-deployed.herokuapp.com/static/#/site/heading

Reply:
@JakeLaCombe @bjankord  by using our header we would require every package to add Header as a dependency. Do we want that?